examples/data/data-stream-with-tag.mjs

```diff
@@ -0,0 +1,66 @@
+/**
+ * Example to subscribe to the data stream API using a tag.
+ */
+import WebSocket from "ws";
+import { loadTags } from "../management/get-tags.mjs";
+import { config } from "dotenv";
+import path from "path";
+config({
+  path: process.env.CONFIG_PATH || path.resolve(process.cwd(), ".env"),
+});
+const wsBaseUrl = process.env.WS_BASE_URL || "wss://api.akenza.io";
+const apiKey = process.env.API_KEY;
+
+const page = await loadTags(0);
+if (page.totalElements == 0) {
+  console.log("no tags found");
+  process.exit(-1);
+}
+const subscriptions = [{ tagId: page.content[0].id, topic: "*" }];
+const ws = new WebSocket(`${wsBaseUrl}/v3/data-streams?xApiKey=${apiKey}`, {
```

Comment:
is this param for api key correct? ^^

Reply:
yes it is correct, but I agree not the most beautiful thing..